src/GuardClauses.UnitTests/GuardAgainsOutOfRange.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Ardalis.GuardClauses;
+using System;
+using Xunit;
+
+namespace GuardClauses.UnitTests
+{
+    public class GuardAgainsOutOfRange
+    {
+        [Fact]
+        public void DoesNothingGivenInRangeValueUsingShortcutMethod()
+        {
+            Guard.AgainsOutOfRange(2, "index", 1, 5);
```

Comment:
To make it more clear that this is an inclusive range, can you add more test cases to cover the full width (or at least the limits, 1 and 5) of the range? You can use a [Theory] and parameterize the test method to pass in the candidate numbers.

Reply:
Hm, seems reasonable. Is it what are you talking about?

```c#
        [Theory]
        [InlineData(1, 1, 5)]
        [InlineData(2, 1, 5)]
        [InlineData(3, 1, 5)]
        [InlineData(4, 1, 5)]
        [InlineData(5, 1, 5)]
        public void DoesNothingGivenInRangeValueUsingShortcutMethod(int input, int rangeFrom, int rangeTo)
        {
            Guard.AgainsOutOfRange(input, "index", rangeFrom, rangeTo);
        }
```

Reply:
Exactly. You could probably reduce a bit if you just show from 1 to 3.